aufs.go

```diff
@@ -109,6 +109,7 @@ func (o *snapshotter) Usage(ctx context.Context, key string) (snapshots.Usage, e
 	}
 	id, info, usage, err := storage.GetInfo(ctx, key)
 	if err != nil {
+		t.Rollback()
```

Comment:
maybe we can just put the rollback before if condition because the following line upperPath doesn’t need to be in transaction